tests/unit/peapods/remote/test_remote.py

```diff
@@ -15,23 +15,24 @@ def test_fetch_files_from_yaml():
         'peas': [
             {
                 'name': 'encode',
-                'uses': 'yamls/encoder.yml',
+                'uses': 'tests/unit/peapods/remote/yamls/encoder.yml',
```

Comment:
use `cur_dir` as reference, since there is no guarantee from which base folder this is going to be run.